docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_exposure/index.rst

```diff
@@ -7,3 +7,172 @@ Iron Deficiency Risk Exposure
+
+  * - MEIDs
+  * - 23376, 23381, 23386, 23377, 23382, 23387, 23378, 23383, 23388, 23379, 23384, 23389, 1929, 1930, 1931, 1925, 1926, 1927, 2131, 2132, 2133, 2065, 2066, 2067, 2082, 2083, 2084, 2113, 2114, 2115, 2506, 2507, 2508, 2119, 2120, 2121, 2492, 2493, 2494, 2495, 2496, 2497, 2498, 2499, 2500, 2502, 2503, 2504, 2475, 2476, 2477, 2478, 2479, 2480, 2481, 2482, 2483, 2485, 2486, 2487, 2489, 2490, 2491, 1666, 1667, 1668, 19391, 19392, 19393, 19395, 19396, 19397, 19399, 19400, 19401, 16314, 16315, 16316, 1538, 1539, 1540, 1522, 1523, 1524, 1532, 1533, 1534, 1476, 1477, 1478, 19737, 19738, 19739, 19798, 19799, 19800, 18864, 18865, 18866, 18861, 18862, 18863
+
+.. note:: 
+
+	The MEIDs listed above were obtained from the *"out_meids"* tab in the excel file hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/in_out_meid_map.xlsx>`_.
+
+Location-Specific TMREL Exposure Estimation
+++++++++++++++++++++++++++++++++++++++++++++
+
+This estimation approach is described in the GBD 2019 Risk Factor Methods Appendix, but NOT used in GBD 2019 iron deficiency risk exposure calculations. Therefore, the following description does not reflect the values stored for the GBD 2019 iron deficiency risk factor, but represent an alternative estimation approach that is "conceptually/clinically more correct" than the Global TMREL exposure estimation approach according to a correspondence with Nick Kassebaum, although it did not meet the GBD 2019 requirement that TMRELs are not location-specific.
+
+The location-specific TMREL exposure estimation approach has the following steps:
+
+#. Estimate a location-specific TMREL, defined as the counterfactual population mean hemoglobin concentration in the absence of all diseases that cause iron deficiency
+
+	:math:`TMREL = mean_\text{Hgb} + \sum_{c=1}^{n} p_c * shift_c`
+
+	Where, :math:`\sum_{c=1}^{n} p_c * shift_c` represents the sum of the cause-specific prevalence multiplied by the cause-specific hemoglobin shift for all causes of iron deficiency and :math:`mean_\text{Hgb}` is the observed mean population hemoglobin concentration (MEID 10487).
+
+	An estimation of the TMREL using this method can be found `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/custom_paf_calculations.ipynb>`_.
+
+	The cause-specific hemoglobin shifts represent the average difference in hemoglobin concentration among individuals afflicted with that cause and the general population (from the literature) and can be found `here <https://github.com/ihmeuw/sim_sci_maternal_anemia/blob/master/data_validation/maternal_disorders_burden/hb_shifts.csv>`_. Note that these shifts are sex-specific but do not vary by age. Additionally, these shifts apply to mean hemoglobin concentration only; it is assumed that there is no change in hemoglobin distribution standard deviation, which is a limitation in this analysis due to a significant data gap.
```

Comment:
Where did you get the list of shifts? Is there a source you can cite?

Reply:
Also, where is the assumption of no change in hemoglobin distribution standard deviation used?  And what are its implications?

Reply:
Came from the iron deficiency modelers and is hosted in their stash repo which is linked

Reply:
The standard deviation assumption is used when applying the cause-specific shifts... it assumes that the entire shape of the hemoglobin distribution is preserved when a given cause of anemia is "removed" from the population, when in reality the shape and standard deviation of the distribution will likely change because hemoglobin observations from the lower end of the distribution will be more likely to be "removed." Does that help clarify? 

I believe (although I am not certain about this) that the implications are that we will underestimate the TMREL because of this. Do you agree?

Reply:
I'm not sure.  If we're just talking about the equation for the TMREL, I don't see any place where the shape of the distribution comes into play, because means are additive regardless of the shape of the distribution.  How exactly do you think the assumption of constant variance will cause the TMREL to be underestimated?

I think if we wanted to compute the entire TMRED (i.e. the theoretical minimum risk distribution, not just its mean), then certainly the variance of this distribution would be smaller than the variance of the overall Hgb distribution before causes of anemia are removed.  But I think to calculate its mean (which is what it looks like they're calling the TMREL), the given equation should work (at least under the assumption that causes of low hemoglobin are mutually exclusive, which seems suspect... not sure if it should still work without that assumption).  Do you know where in their methodology the standard deviation(s) of the distribution(s) get used?

Reply:
Oh good point, I think you're right.

So it's a continuous risk model, so when calculating the PAF, they use the ensemble distribution of individual values within a population compared to the TMREL. I agree that this will not matter when calculating the TMREL as a point value, but when a counterfactual exposure distribution is assumed, the constant variance assumption will impact the PAF calculation, I believe.

I'll update this to try to make it more clear.

Reply:
Oh good point, I think you're right. I will update this.